src/java.base/share/classes/java/util/AbstractMap.java

```diff
@@ -598,6 +598,9 @@ private static boolean eq(Object o1, Object o2) {
      * arrays of {@code SimpleEntry} instances in method
      * {@code Map.entrySet().toArray}.
      *
+     * @param <K> the type of keys maintained
```

Comment:
Please update to match java.util.Map, which says "the type of keys maintained by this map"

Reply:
I said "keys maintained", omitting "by this map" to finesse the question of if the SimpleEntry class *is* a map, or is used to implement a map, etc. I can change it to include "by this map" if the map/entry distinction is okay to be blurred.

Reply:
Whoops, sorry, this is `SimpleEntry`, which is _not_ a `Map`. In this case I'd follow `Map.Entry` which says "the type of the key" and "the type of the map".

Reply:
Will change to the Map.Entry wording "the type of key" and "the type of the value", respectively.